<?xml version="1.0"?>

<launch>

    <arg name="vehicle_frame_id" value="cmu_rc2_sensor"/> 
    <arg name="fixed_frame_id" value="cmu_rc2_sensor_init"/>                   
    <node pkg="trajectory_library" type="TrajectoryLibraryWrapper" name="TrajectoryLibraryWrapper" output="screen">

        <param name="odometry_topic" value="/cmu_rc2/integrated_to_init"/>
        <param name="costmap_topic" value="/voxel_grid/obstacle_map" />

        <param name="vehicle_frame" value="$(arg vehicle_frame_id)"/>
        <param name="base_frame" value="$(arg fixed_frame_id)"/>

        <param name="trajectory_library_filepath" value="$(find trajectory_library)/config/paths.ply"/>
        
        <param name="min_x" value="-50.0"/>
        <param name="min_y" value="-50.0"/>
        <param name="resolution" value="0.1"/>
        <param name="width" value="1000"/>
        <param name="height" value="1000"/>


    </node>

</launch>



<!-- launch-prefix="cuda-gdb -ex 'break main' -ex 'run'" -->